web/settings.py

```
@@ -363,3 +363,39 @@
+if DEBUG:
+    INSTALLED_APPS.append("debug_toolbar")
+    MIDDLEWARE.insert(0, "debug_toolbar.middleware.DebugToolbarMiddleware")
+    INTERNAL_IPS = ["127.0.0.1"]
+
+TEST_RUNNER = "django.test.runner.DiscoverRunner"
+
+if "test" in sys.argv:
+    CACHES = {
+        "default": {
+            "BACKEND": "django.core.cache.backends.dummy.DummyCache",
+        }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)